config/settings.yml

```diff
@@ -39,3 +39,4 @@ provlog:
   enable: false
 
 workflow_services_url: ''
+c2m_sql_limit: 1000
```

Comment:
are we ok with this simply being set here, or should it also go in shared_configs?   (If you already have PRs for shared_configs, you need to paste the urls in the channel for us to see them.)

Reply:
It might be good to have them also in shared_configs, if we ever want to alter the limit between deployed environments, for example. I'll set that up.